### ISSUE TITLE
Add lang-open command for languages.toml

### ISSUE THREAD

**Issue by AceofSpades5757** [helix-editor/helix#3206]
#### Describe your feature request

Add a `lang-open` command, just like the `config-open` and `log-open` commands, to open the platform-specific config file.

If this isn't already in the works (I didn't see any related issues or pull requests), then I have the bandwidth to make a pull request.

**Comment by the-mikedavis** [comment]
The language config is merged from `.helix/languages.toml`, `~/.config/helix/languages.toml` and the compiled-in `languages.toml` at the root of the repo.  Because of the merging behavior I don't think it makes sense to have a `:lang-open` command. If anything it might make sense to have it open a picker with up to all three of those files but that would most likely just be confusing and these `:*-open` commands are meant to be as simple and straightforward as possible. Plus you can't edit the compiled-in `languages.toml`

**Comment by AceofSpades5757** [comment]
It would still be nice to have a nice, built-in way to open the user/local languages.toml file.

**Comment by AceofSpades5757** [comment]
Those are really good points though. What about a config picker for all config files, `~/.config/helix/config.toml`, `~/.config/helix/languages.toml`, `.helix/languages.toml`, and any future config files that get added to helix? For the future configs, I was thinking there may be individual configuration files based on language, so you may just want a picker to quickly edit the right one. If needed, users can always add them to their `config.toml` files as a keymapping.

**Comment by AceofSpades5757** [comment]
Could also `config-open` to default go to `~/.config/helix/config.toml`, and give it an auto-completer for any other, relevant config files as a second argument.

e.g. `config-open` does the same thing it does currently.
e.g. `config-open ~/.config/helix/languages.toml` would be an auto-complete option.

Could instead give users named files, like `User Languages Config`, `Project Languages Config` (`.helix/languages.toml`), etc.

**Comment by the-mikedavis** [comment]
`:config-open` and `:log-open` are meant as very beginner-friendly ways to open up files so you don't need to read the documentation to know where they are. Language config is more complicated: you need to read the language config documentation to understand the behavior.

Optimizing how easily we can open config files seems like hair-splitting to me. Instead I think it would be better to improve the in-editor documentation of configuration. The idea is half-baked but it would be nice to have a man-page that explains what each file does and covers the merging behavior and then you could `gf` on the paths it has to open the files.

**Comment by AceofSpades5757** [comment]
The new issue title and description should, hopefully, lead to something better.

**Comment by willparsons** [comment]
> Those are really good points though. What about a config picker for all config files, `~/.config/helix/config.toml`, `~/.config/helix/languages.toml`, `.helix/languages.toml`, and any future config files that get added to helix? For the future configs, I was thinking there may be individual configuration files based on language, so you may just want a picker to quickly edit the right one. If needed, users can always add them to their `config.toml` files as a keymapping.

I agree with the idea of a config picker, it keeps the process simple and prevents the need for more commands.

**Comment by arcstur** [comment]
I think having a command to open a language config file is useful.

You said there are three files, but that could be expanded to the config.toml as well. I mean, Helix has a default configuration, which isn't expressed by a config.toml, but could be thought as if it was. And, in the future, adding support for a `.helix/config.toml` could be a good idea.

For newcomers it is useful to have a command to readily open the proper global config file. The command description, just as with the `config-open` description, can make it clear "Open the user language.toml file", just like "Open the user config.toml file".
